test/passing/source.ml.ref

```diff
@@ -3502,8 +3502,7 @@ let ssmap =
   )
 
 let ssmap :
-    (module
-     MapT
+    (module MapT
```

Comment:
It wouldn't be bad to have an additional test where `MapT` is long enough to break, to keep track of what happens in that case.

Reply:
Added.